wormchain/Makefile

```diff
@@ -44,17 +43,13 @@ validators:
 build/wormchaind: cmd/wormchaind/main.go $(GO_FILES)
 	go build -v $(BUILD_FLAGS) -tags ledger -o $@ $<
 
-check-ignite:
-	@ if [ "$(shell ignite version | awk '/Ignite CLI version:/ { print $$4 }')" != "$(IGNITE_EXPECTED_VERSION)" ] ; then echo "Expected ignite version $(IGNITE_EXPECTED_VERSION)" && exit 1 ; fi
-
-proto: check-ignite $(PROTO_FILES)
-	ignite generate proto-go
-	touch proto
+proto: $(PROTO_FILES)
+	cd .. && docker build --target go-export -f wormchain/Dockerfile.proto -o type=local,dest=wormchain .
```

Comment:
do you need to `cd ..` as opposed to 
```
docker build --target go-export -f Dockerfile.proto -o type=local,dest=. ..
```

Reply:
we need to `cd ..`. `docker build` only has access to the directories in the context it is run.

Reply:
Yes, I am suggesting you run it from the context of `..`

Reply:
My bad the above does work.. will update PR.